### PR TITLE
[feat](profile) Print changed session var in profile

### PR DESCRIPTION
**Pull request by zhiqiang-hhhh**
Add changed session var like follow:
```
Changed Session Variables:
VarName                                         | CurrentValue        | DefaultValue
------------------------------------------------|---------------------|-------------
character_set_results                           | NULL                | utf8mb4
enable_profile                                  | true                | false
enable_auto_analyze                             | false               | true
max_scan_key_num                                | -1                  | 48
enable_prepared_stmt_audit_log                  | true                | false
sql_mode                                        | STRICT_TRANS_TABLES |
wait_timeout                                    | 31000               | 28800
parallel_pipeline_task_num                      | 8                   | 0
num_scanner_threads                             | 10                  | 0
experimental_enable_agg_state                   | true                | false
max_pushdown_conditions_per_column              | -1                  | 1024
experimental_parallel_scan_min_rows_per_scanner | 16384               | 2097152
max_allowed_packet                              | 1048576             | 16777216
```

### CODE REVIEW

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/common/profile/Profile.java [hunk 1]
```diff
@@ -363,49 +366,13 @@ public String getProfileBrief() {
...
 
-    // Read file if profile has been stored to storage.
+    // Return if profile has been stored to storage
     public void getExecutionProfileContent(StringBuilder builder) {
         if (builder == null) {
             builder = new StringBuilder();
         }
 
         if (profileHasBeenStored()) {
-            LOG.info("Profile {} has been stored to storage, reading it from storage", id);
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/common/profile/Profile.java:373]: 为什么改这个？

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Review by yiguolei — APPROVED** [review]

**Review by zclllyybb — APPROVED** [review]
LGTM

**Review by jacktengg — APPROVED** [review]

**Pull request merged**